### PR TITLE
Making changes to Contributing.MD

### PR DESCRIPTION
**Pull request by seun-beta**
I rephrased some words and added some extra information about contributing to CHAOSS. 
My opinion: I think some headers should be of smaller sizes to conform to visual hierarchy. I was also thinking of making the file a little friendly by adding emojis.
Do you think pictures should be added to help explain some points?
I would love to know if the changes I made are good to go so I can continue making changes.
Thank you

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
+## Who can contribute?
 
-Anyone can contribute to CHAOSS on any of our communication channels. See <https://chaoss.community/participate/>.
+Anyone can contribute to CHAOSS on any of our communication channels. See <https://chaoss.community/participate/> to learn more about how to participate.
 
 ## Which channel should I use?
+CHAOSS uses the following channels for contributions:
+- Mailing Lists
+- Community calls
+- Working group calls.
```
Comment (GeorgLink) [inline CONTRIBUTING.md:16]:
```suggestion
- Working group calls
- GitHub
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
+Bug reports and specific feature requests are best discussed in an issue on the repository they pertain to. You can do this by creating a issue and explaining what bug you found in the comment section. The conversation moves from there.
 
-### Code or document change contributions (GitHub interface)
+### Code or document change contributions (GitHub User Interface)
 
 Changes can be made from within the GitHub user interface. As a non-maintainer, GitHub will create a fork under your name and ask that you create a pull request. The pull request lets the repository maintainers know of your change and provides a space to discuss the change.
 
 Every commit needs a [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
-A DCO sign-off is accomplished by writing `Signed-off-by: Your Name <YourName@example.org>` into the commit comment field. This can be automated by using a browser plugin like [scottrigby/dco-gh-ui](https://github.com/scottrigby/dco-gh-ui)
+A DCO sign-off is accomplished by writing into the commit comment field.
```
Comment (GeorgLink) [inline CONTRIBUTING.md:31]:
```suggestion
A DCO sign-off is accomplished by writing into the commit comment field:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
-### Code or document change contributions (GitHub interface)
+### Code or document change contributions (GitHub User Interface)
 
 Changes can be made from within the GitHub user interface. As a non-maintainer, GitHub will create a fork under your name and ask that you create a pull request. The pull request lets the repository maintainers know of your change and provides a space to discuss the change.
 
 Every commit needs a [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
-A DCO sign-off is accomplished by writing `Signed-off-by: Your Name <YourName@example.org>` into the commit comment field. This can be automated by using a browser plugin like [scottrigby/dco-gh-ui](https://github.com/scottrigby/dco-gh-ui)
+A DCO sign-off is accomplished by writing into the commit comment field.   
+`Signed-off-by: Your Name <YourName@example.org>`   
+It should be noted that the angled brackets **< >** are to be ingored when typing your email address.
```
Comment (GeorgLink) [inline CONTRIBUTING.md:33]:
```suggestion
It should be noted that the angled brackets `<` and `>` need to be included with the email.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,29 +1,38 @@
...
+ 
+This process can be automated by using a browser plugin like [scottrigby/dco-gh-ui](https://github.com/scottrigby/dco-gh-ui). This plugin is non intrusive and works only on GitHub. Yu would need to input your registered GitHub email address in the email field.
```
Comment (GeorgLink) [inline CONTRIBUTING.md:35]:
```suggestion
This process can be automated by using a browser plugin like [scottrigby/dco-gh-ui](https://github.com/scottrigby/dco-gh-ui). This plugin is non intrusive and works only on GitHub. You would need to input your registered GitHub email address in the email field.
```

**Review by seun-beta — COMMENTED** [review]
Made the necessary changes. Thanks a lot.

**Comment by seun-beta** [comment]
Hello @GeorgLink I think the PR is ready.

**Review by GeorgLink — APPROVED** [review]
Thanks @seun-beta

**Pull request merged**